person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_reader :id # getter
+  attr_accessor :name, :age # getter & setter
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @parent_permission = parent_permission
+    @id = Random.rand(1..1000)
+    @name = name
+    @age = age
+  end
+
+  def can_use_services?
+    of_age? || @parent_permission ? true : false
```

Comment:
- [ ] This code can be improved. The `@parent_permission` instance variable will already be `true` or `false` depending on how an object is instantiated, so we don't really need to check for true or false here anymore. We can simply write:

  ```
  of_age? || @parent_permission
  ```